Resource/PYTHON/multilateration/multilateration.py

```diff
@@ -0,0 +1,203 @@
+"""Estimates position of sound source in xy-coordinates based on time difference of arrival between hydrophones.
+"""
+
+
+import math
+
+import numpy as np
+
+import multilateration.parameters as param
+
+
+# Initializing the variables maximum_time_diff and max_hydrophone_distance.
+# These values are updated in the function initialize_trilateration_globals.
+max_hydrophone_distance = None
+max_time_diff = None
+
+
+def calculate_distance(hyd_1_x, hyd_1_y, hyd_1_z, hyd_2_x, hyd_2_y, hyd_2_z):
+    return float(
+        math.sqrt(
+            math.pow(
+                hyd_2_x - hyd_1_x,
+                2,
+            )
+            + math.pow(
+                hyd_2_y - hyd_1_y,
+                2,
+            )
+            + math.pow(
+                hyd_2_z - hyd_1_z,
+                2,
+            )
+        )
```

Comment:
```suggestion
        np.sqrt(
            (hyd_2_x - hyd_1_x) ** 2
            + (hyd_2_y - hyd_1_y) ** 2
            + (hyd_2_z - hyd_1_z) ** 2
        )
```
Then you can remove `import meth`